### ISSUE TITLE
Upgrade several minor versions to Harbor 2.6.2

### ISSUE THREAD

**Issue by dan-m8t** [goharbor/harbor#17786]
Hey,

currently I am using Harbor 2.5.1 (Docker setup) and I want to update to Harbor 2.6.2.

As far as I know it is necessary to step by step because of the postgresql schema migrations listed [here](https://github.com/goharbor/harbor/tree/main/make/migrations/postgresql)

Am I assuming right that I need to update like this:

2.5.1 -> 2.5.2 -> 2.5.3 -> 2.6.0 -> 2.6.2 ?

I already have experience with updates where I skipped several minor versions, those failed because of the missing psql migrations, so I went the way of following the path of postgresql migrations, which is kinda alot of work, because I cannot update everytime a new Harbor version is available and I'm living in a disconnected environment, which means I need to download all the offline installers, unpack, load them to docker, run the migration script etc, you get it :) 

Is there a workaround for at least the 2.5.x minor versions? Like can I apply the postgresql migrations from 2.5.2 and 2.5.3 without going full update cycle? How could I do this? Simply with interacting via psql with the database I guess?

Thanks

**Comment by MinerYang** [comment]
Normally, we do support at least v-2  minor version upgrading and migration, for example you can upgrade from v2.5.1 to v2.6.2 directly. You can refer to bellowing doc.
https://goharbor.io/docs/2.6.0/administration/upgrade/

If it's not work, could you provide the migrate/upgrade steps and  the harbor.yml(configuration) files(for database are you using an external one or internal one) ?

**Comment by dan-m8t** [comment]
I'll test this out on a test setup and report back. My last approaches to do kind of big steps to update all failed because of the missing psql schemas.

**Comment by dan-m8t** [comment]
OK I tested on a local setup to upgrade from 2.5.1 to 2.6.2 and that worked just fine.

I saw this in the logs:

```
2022-11-14T08:53:56Z [INFO] [/vendor/github.com/golang-migrate/migrate/v4/migrate.go:765]: 81/u 2.5.2_schema (15.5109ms)
2022-11-14T08:53:56Z [INFO] [/vendor/github.com/golang-migrate/migrate/v4/migrate.go:765]: 82/u 2.5.3_schema (30.3643ms)
2022-11-14T08:53:56Z [INFO] [/vendor/github.com/golang-migrate/migrate/v4/migrate.go:765]: 90/u 2.6.0_schema (83.4847ms)
2022-11-14T08:53:56Z [INFO] [/vendor/github.com/golang-migrate/migrate/v4/migrate.go:765]: 91/u 2.6.2_schema (97.3885ms)
```

Afaik this was not implemented when I updated from 2.4.X to 2.5.1, so everythings should be fine for my production update :)

Thanks for clarification, I close the ticket.
